README.rst

```diff
@@ -0,0 +1,98 @@
+E3SM Data Documentation
+=======================
+
+TODO: Add an overview here
+
+
```

Comment:
TODO

Need some content to put in the overview for this repo.

Reply:
Maybe @golaz can handle this?

Reply:
We can probably do that in a future issue/pull request.

Reply:
Got it